common/test/acceptance/pages/common/__init__.py

```diff
@@ -1,10 +1,14 @@
+"""
+Get the URL of the instance under test
```

Comment:
Hm, I don't think this should be the docstring. I would keep the comment where it was and add a docstring that instead says something like
```
"""
Package of common page objects for acceptance tests
"""
```

Reply:
Docstring updated :)